src/main/scala/com/snowflake/snowpark/Column.scala

```diff
@@ -718,6 +718,196 @@ case class Column private[snowpark] (private[snowpark] val expr: Expression) ext
   }
 
   protected def withExpr(newExpr: Expression): Column = Column(newExpr)
+
+  /**
+   * Function that validates if the value of the column is
+   * within the list of strings from parameter.
+   * @param strings List of strings to compare with the value.
+   * @return Column object.
```

Comment:
Can you add `@since 1.10.0` in all functions' docs? Next release will be `1.10.0`.
We use this way to track when the functions introduced.